commands/audit/scarunner.go

```diff
@@ -51,74 +63,34 @@ func buildDepTreeAndRunScaScan(auditParallelRunner *utils.SecurityParallelRunner
 	if err != nil {
 		return
 	}
-
-	scans := getScaScansToPreform(auditParams)
-	if len(scans) == 0 {
+	if !shouldPreformScaScan(cmdResults) {
 		log.Info("Couldn't determine a package manager or build tool used by this project. Skipping the SCA scan...")
```

Comment:
error message doesn't fit for what the shouldPreformScaScan function is checking